### ISSUE TITLE
idea: spaces represent slashes

### ISSUE THREAD

**Issue by jjb** [topfunky/PeepOpen-Issues#76]
given that:
- peepcode's main strength/distinction is the ability to search for directories in addition to file names
- the behavior of inserting slashes is pretty perfect and super handy (can instantly get to index.html.erb for Users with v/u/i)
- spaces currently have no function other the visually aiding in the construction of longer searches

how about having spaces represent slashes? no need for any fancy visualization or explanation -- entering a space would just enter a slash, plain and simple.

**Comment by topfunky** [comment]
That's an interesting idea!

A previous feature request was from a person who used spaces instead of underscores.

Spaces are currently ignored, so you could enter a space where a slash should be and it might match the right thing anyway.

**Comment by jjb** [comment]
Yep, experimented with that a bit, slashes are always much much more precise, and make for a nice mental map to get to the file.

True, making the spaces act as slashes will conflict with expectations from TextMate command-t… but isn't differentiation and improvement the point? :)

**Comment by jjb** [comment]
(it could be an option in preferences...)

**Comment by Soleone** [comment]
+1 for this idea. I find it kind of cumbersome to manually type the slash, using space is a lot faster imo.

**Comment by jweir** [comment]
I use a modified FuzzyFinder in VIM which maps spaces to 
"**/" and it works great.
